components/polylith/diff/collect.py

```diff
@@ -75,13 +75,30 @@ def get_latest_tag(root: Path, key: Union[str, None]) -> Union[str, None]:
     return next((tag for tag in res.stdout.decode("utf-8").split()), None)
 
 
-def get_files(tag: str) -> List[Path]:
+def from_root(root: Path, path: Path) -> Path:
+    if root.name not in path.parts:
+        return path
```

Comment:
This is a bit risky, if the root name collides with anything in the workspace. Need to re-think this.

Reply:
Having the command know if the Polylith Workspace is in a subfolder or not would be good.